Sources/ShopifyAcceleratedCheckouts/Wallets/ApplePay/ApplePayButton.swift

```diff
@@ -146,7 +146,7 @@ struct Internal_ApplePayButton: View {
 
 /// Used to set the label of the Apple Pay button
 /// see `.applePayLabel(label:)`
-public enum ApplePayButtonLabel {
+public enum ApplePayButtonLabel: CaseIterable {
```

Comment:
```suggestion
public enum ApplePayButtonLabel: CaseIterable, String {
```

If we extend from String too we can remove the initializer as you can call `ApplePayButtonLabel(rawValue: "SomeString")`